test/unit-test/FreeRTOS_DNS_Cache/FreeRTOS_DNS_Cache_utest.c

```diff
@@ -630,7 +632,7 @@ void test_prepare_DNSLookup_IPv6( void )
     BaseType_t x = 0U;
     BaseType_t xFamily;
     struct freertos_addrinfo * pxAddressInfo = &pucAddrBuffer[ 0 ];
-    struct freertos_addrinfo ** ppxAddressInfo = &pucAddrBuffer[ 1 ];
+    struct freertos_addrinfo ** ppxAddressInfo = ( struct freertos_addrinfo ** ) &pucAddrBuffer;
     IPv46_Address_t xAddress;
 
     xFamily = FREERTOS_AF_INET6;
```

Comment:
[line: 643:](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1090/files#diff-aaa8da95d9503fb80decbecf87cb0ed4d35cb3c1242e3e829c4e7130c928c3adR643) 
`struct freertos_addrinfo * pxAddrInfo = &pucAddrBuffer[ 2 ];`  

is accessing an out of bound array index. Could you help in updating that?